symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/auth/model/TokenPair.java

```diff
@@ -0,0 +1,26 @@
+package com.symphony.bdk.app.spring.auth.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+
+/**
+ * A pair of tokens used during the Circle of Trust authentication.
+ */
+@Data
```

Comment:
Same here as for UserId can we make it immutable?

Reply:
if it is immutable, we cannot deserialize it anymore?

Reply:
https://www.baeldung.com/jackson-deserialize-immutable-objects

Reply:
https://www.google.com/search?q=jackson+immutable+objects :)

Reply:
OOOOPPPSSS! 😴